docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+
+services:
+  chrono:
+    image: "chrono"
+    restart: on-failure
+    ports:
+      - 3000:5000
+    env_file:
+      - .env
```

Comment:
```yaml
depends_on:
  mongo
```
imo adding this would be good [ref](https://docs.docker.com/compose/compose-file/05-services/#depends_on).

Also 
```yaml
ports:
  - <port>:<port>
```
Exposes your port. You don't have to `EXPOSE`  explicitly again in `Dockerfile`